opentargets.json

```diff
@@ -1323,11 +1323,13 @@
     "studyStartDate": {
       "type": "string",
       "format": "date",
-      "description": "Start date of study in a YYYY-MM-DD format"
+      "description": "Start date of study in a YYYY-MM-DD format",
+      "pattern": "(((19|20)([2468][048]|[13579][26]|0[48])|2000)[-]02[-]29|((19|20)[0-9]{2}[-](0[469]|11)[-](0[1-9]|[12][0-9]|30)|(19|20)[0-9]{2}[-](0[13578]|1[02])[-](0[1-9]|[12][0-9]|3[01])|(19|20)[0-9]{2}[-]02[-](0[1-9]|1[0-9]|2[0-8])))"
     },
     "studyStopReason": {
       "type": "string",
-      "description": "Reason why a study has been stopped"
+      "description": "Reason why a study has been stopped",
+      "pattern": "^[^\\ ].*[^\\ ]$"
```

Comment:
Why is this pattern required?

Reply:
To avoid any leading or trailing white spaces or newlines in this field.

Reply:
Ah, yes, these weird whitespaces.